src/core/CompUnit/Repository/Installation.pm

```diff
@@ -357,8 +361,8 @@ sub MAIN(:$name is copy, :$auth, :$ver, *@, *%) {
         my $prefix = self.prefix;
         my $lookup = $prefix.child('short').child(nqp::sha1($name));
         if $lookup.e {
-            my $version = self!repository-version;
-            my @dists = $version < 1
+            my $repo-version = self!repository-version;
+            my @dists = $repo-version < 2
```

Comment:
This needs to stay a 1. In version 0 $lookup is a file, so we read it using $lookup.lines. In version 1 and up it's a directory, so we use $lookup.dir